src/test/librbd/fsx.cc

```diff
@@ -2393,12 +2393,16 @@ test(void)
 			log4(OP_SKIPPED, OP_WRITESAME, offset, size);
 			goto out;
 		}
-        case OP_COMPARE_AND_WRITE:
-                /* compare_and_write not implemented */
-                if (!ops->compare_and_write) {
-                        log4(OP_SKIPPED, OP_COMPARE_AND_WRITE, offset, size);
-                        goto out;
-                }
+		break;
+	case OP_COMPARE_AND_WRITE:
```

Comment:
@scienceluo Yeah, I was wondering why the whole block shows up in green. But the resulting code was correct.